mqtt/mqtt-broker/src/translation.rs

```diff
@@ -0,0 +1,81 @@
+use crate::ClientEvent;
```

Comment:
As a quick check of an idea, this may be good, but in order to check it in the master, this code is to be oxidized :)